test.js

```diff
@@ -50,6 +54,33 @@ describe('pixabayjs', function() {
     var query;
 
     before(function(done) {
+      mockagent.target(superagent);
+
+      mockagent.url(pixabayUrl, function(res) {
+        var query = url.parse(this.url).query;
+        var queryParams = querystring.parse(query);
+
+        var firstPage = queryParams.page !== '2';
```

Comment:
Can we dumb down this logic a little. It is a little hard to parse. Would this work:

```javascript
if (queryParams.page === 1) {
  responseBody = [0, 1, 2, 3];
} else if (queryParams.page === 2) {
  responseBody = [4, 5];
} else {
  throw new Error('Mock encountered an unexpected condition');
}
```